tests/pds/dataservice_test.go

```diff
@@ -1300,3 +1301,55 @@ var _ = Describe("{RollingRebootNodes}", func() {
 		defer EndTorpedoTest()
 	})
 })
+
+var _ = Describe("{AddAndValidateUserRoles}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("AddAndValidateUser", "Add users with admin or non admin privileges and validate the same.", nil, 0)
+	})
+
+	It("add users with admin or non admin privileges and validate.", func() {
+		usersParam := params.Users
+
+		Step("Adding users having admin previledges and validating for same.", func() {
+			log.Info("Adding user with admin privileges.")
+			username := usersParam.AdminUsername
+			password := usersParam.AdminPassord
+			log.InfoD("username - %v", username)
+			pdslib.Components.AccountRoleBinding.AddUser(accountID, username, true)
```

Comment:
validate the response, like you did for `ListAccountsRoleBindings`

Reply:
Done